src/ai-bundle/src/Command/ChatCommand.php

```diff
@@ -0,0 +1,189 @@
+    {
+        $this
+            ->addArgument('agent', InputArgument::OPTIONAL, 'The name of the agent to chat with')
+            ->setHelp(
+                <<<'HELP'
+                The <info>%command.name%</info> command allows you to chat with different agents.
+
+                Usage:
+                  <info>%command.full_name% [<agent_name>]</info>
+
+                Examples:
+                  <info>%command.full_name% wikipedia</info>              # Use short name
+                  <info>%command.full_name% ai.agent.wikipedia</info>     # Use full service ID
+
+                If no agent is specified, you'll be prompted to select one interactively.
+                
+                The chat session is interactive. Type your messages and press Enter to send.
+                Type 'exit' or 'quit' to end the conversation.
+                HELP
+            );
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        // Build a map of available agents with proper names
+        $availableAgents = [];
+        $agentMap = [];
+
+        // Get all provided service IDs from the locator
+        $serviceIds = $this->agents->getProvidedServices();
+
+        foreach (array_keys($serviceIds) as $serviceId) {
+            // Extract the agent name from the service ID (e.g., 'ai.agent.wikipedia' -> 'wikipedia')
+            if (str_starts_with($serviceId, 'ai.agent.')) {
+                $name = substr($serviceId, \strlen('ai.agent.'));
+            } else {
+                $name = $serviceId;
+            }
+
+            $availableAgents[$name] = $name;
+            $agentMap[$name] = $this->agents->get($serviceId);
+            // Also map the full service ID to support both formats
+            $agentMap[$serviceId] = $this->agents->get($serviceId);
+        }
+
+        if (0 === \count($availableAgents)) {
+            throw new InvalidArgumentException('No agents are configured.');
+        }
+
+        $agentArg = $input->getArgument('agent');
+        $this->agentName = \is_string($agentArg) ? $agentArg : '';
+
+        // If no agent specified or invalid agent, prompt for selection
+        if (!$this->agentName || !isset($agentMap[$this->agentName])) {
+            if ($this->agentName && !isset($agentMap[$this->agentName])) {
+                $io->warning(\sprintf('Agent "%s" not found.', $this->agentName));
+            }
+
+            // Interactive selection
+            $question = new ChoiceQuestion(
+                'Please select an agent to chat with:',
+                array_keys($availableAgents),
+                0
+            );
+            $question->setErrorMessage('Agent %s is invalid.');
+
+            /** @var \Symfony\Component\Console\Helper\QuestionHelper $helper */
+            $helper = $this->getHelper('question');
+            $selectedAgent = $helper->ask($input, $output, $question);
```

Comment:
The best practice is to move the interactive question in the [interact method](https://github.com/symfony/symfony/blob/7278aa035af69b83d398b7220669ecb61be5a639/src/Symfony/Component/Console/Command/Command.php#L242-L244). And make the `agent` argument required.